### ISSUE TITLE
第67回 AWS Expert Online これが基本のマルチアカウント管理 - 定期的に集客する

### ISSUE THREAD

**Issue by PharaohKJ** [jaws-ug-kanazawa/jaws-ug-kanazawa.github.io#214]
- [x] JAWS-UG 金沢 slack
- [ ] JAWS-UG 本家 slack
- [ ] JAWS-UG 本家 slack \#jawsug-chubu
- [ ] twitter(個人アカウントでよい)
- [ ] facebook(公式ページのシェアなどでお願いします)
